docs/tools/offline/cli-test-debug.md

```diff
@@ -2,6 +2,8 @@
 
 Use the `mbed test` command to compile and run tests.
 
+There are two testing frameworks: Greentea and Icetea. Greentea provides tests designed for driver porting and target verification. Icetea provides and manages tests for multiple devices under the test (DUTs) at the same time. For example, you can test the network setup for a server and multiple clients, and simultaneously control them from the test environment.
+
```

Comment:
What do you mean by 'devices under the test (DUTs)'?  Is this just what devices that are currently being tested are called?  The sentence is repetitive, so I would remove this or make a new sentence explaining this.  So: "Icetea provides and manages tests for multiple devices at the same time.  Devices being tested are referred to as devices under test (DUTs)."

Reply:
DUT is acronym for device under test. I don't see the sentence being repetitive.

Reply:
You're saying it's testing devices under test - that's repetitive.  That's why I think it would help to break it into two sentence.

Reply:
Ok, but the second sentence is similarly repetitive now, I don't see the point.

Reply:
The point it to spread out how many times you're repeating 'testing' in the same sentence.  If you want to explain the acronym DUTs it would be better in a separate sentence.

Reply:
I agree with @jen3andruska. Also, the DUT acronym isn't used later in this document, so how about we just leave that out? So just shorten it to "Icetea provides and manages tests for multiple devices at the same time."

Reply:
True, it is defined mostly because the icetea tool itself uses the acronym.